projects/js-packages/publicize-components/src/store/selectors.js

```diff
@@ -237,6 +262,19 @@ export function twitterCardIsCached( state, url ) {
 	return !! state.twitterCards[ url ];
 }
 
+/**
+ * Gets the text that will be used when sharing this post on LinkedIn.
+ *
+ * @returns {string} The share message.
+ */
+export function getTextForLinkedIn() {
+	const { getEditedPostAttribute } = select( 'core/editor' );
+	const meta = getEditedPostAttribute( 'meta' );
+	const text = meta?.jetpack_publicize_message || getEditedPostAttribute( 'title' ) || '';
+
+	return text;
+}
```

Comment:
There's the [useSocialMediaMessage](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/publicize-components/src/hooks/use-social-media-message) hook which returns the message that will be used for sharing—maybe that can be used instead?

It uses the [getShareMessage](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/publicize-components/src/store/selectors.js#L245) selector from the Publicize store which also falls back to the post title.

Reply:
Oh, I didn't know about that. Thanks.